_posts/2020-05-07-report-deep-sea-robotics.md

```diff
@@ -0,0 +1,182 @@
+
+The giant squid (Figure 10) is an elusive deep sea creature that is very rarely captured on camera. Some adaptions that it has to the deep sea include its large eye and lack of bones. Due to the darkness deep underwater, their huge retinas allow them to capture the smallest amount of visible light (Ames, 2019). For a deep sea robot, any pictures or videos, must have its own light source or not use visible light. The giant squid is an invertebrate and lacks bones, very similar to the blobfish’s adaption to the deep sea. 
+
+![figure13](/assets/2020-05-07-report-deep-sea-robotics/13GiantSquid.jpg)
+
+Finally, the Cuvier’s beaked whale can rapidly adjust to the deep sea pressure, diving over 2,000 meters below the sea surface. Unlike other whales and mammals that dive deep beneath the sea, such as the sperm whale, the beaked whale is relatively small (Amos, 2014). 
+
+![figure14](/assets/2020-05-07-report-deep-sea-robotics/14sizeComparisonBetweenSpermWhaleandBeakedWhale.jpg)
+
+In order to dive so deep, the whale must store enough oxygen for the long trips and endure the crushing pressure of the deep sea. Beaked whales rely on oxygen to breathe, so they must be able to hold their breath for long periods of time. In an aquatic robot, this is not a problem because robots do not breathe. As the beaked whales descends, their lungs shrink and gasses such as nitrogen in their blood and tissues will be compressed. The theory to how the beaker whale adapts to this scenario is by collapsing its lungs to force air away from the alveoli, which transfers gasses into the blood (Pavid, 2017). For an aquatic robot, this may mean having collapsible parts. 
+
+### Prototype 1
+To create my initial model and its meshes, I used Blender, a 3D software program. It took me a while to get comfortable with the controls and build 3-D shapes. While deciding what shape to use, I studied different shapes that comprise manmade underwater vehicles such as submarines, cameras, and more. The first prototype was relatively simple and had no device of movement (Figure 12). The shape at the end became a hemisphere on top of protruding sheets that make an ellipsoid (Figure 13). 
+
+![figure15](/assets/2020-05-07-report-deep-sea-robotics/15Prototype1movingDownInDeepSeaEnvironment.gif)
+![figure16](/assets/2020-05-07-report-deep-sea-robotics/16EllipsoidofPrototype2.png)
+
+
+This prototype was modeled as a sdf, a file type in XML format created as part of the Gazebo robot simulator. The sdf file contains objects and/or environments. Some advantages of using the sdf file type are its simplicity and compatibleness with Gazebo Simulator. However, the sdf file type has limitations including the inability to adapt pressure sensors and movement. I moved to the urdf and xacro file formats for Prototype 2, because there is more documentation on how to use urdf files when modeling robots and it has more capabilities such as adding sensors and movement.  
```

Comment:
```suggestion
I modeled this prototype as an SDF file, which is an XML file type created as part of the Gazebo robot simulator. The SDF file contains objects and/or environments. Some advantages of using the SDF file type are its simplicity and compatibleness with Gazebo Simulator. However, the SDF file type has limitations including the inability to adapt pressure sensors and movement. I moved to the URDF and Xacro file formats for Prototype 2, because there is more documentation on how to use URDF files when modeling robots and it has more capabilities such as adding sensors and movement.  
```